Eastern-Research-Group/expert-query
# Feature/310 attains table updates

## Related Issues:
* [EQ-309](https://jira.epa.gov/browse/EQ-309)
* [EQ-310](https://jira.epa.gov/browse/EQ-310)

## Main Changes:
* Updated attains table definitions.
* Moved create table queries and other similar configs to the `tableConfig.json` file.

## Steps To Test:
1. Verify the config updates look correct
2. Ensure env files are pointed at local db
3. Run the etl
4. Start the app
5. Run some queries to test the new and updated columns
